weather_dl/download_pipeline/parsers.py

```diff
@@ -492,4 +494,7 @@ def validate_all_configs(configs: t.List[Config]) -> None:
     require(all_equal(clients), f'All configs must request data from the same client, {clients[0]!r}.')
 
     kwargs = [conf.kwargs for conf in configs]
+    # Since the config file names will vary between configs,
+    # there's no need to perform an equality check for this.
```

Comment:
Thanks for the comment!

Reply:
Thanks. Though this code has been removed since we moved the `config_name` property to the top level of the config dataclass.